tests/integration/targets/systemd_creds_decrypt/aliases

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+needs/root
+
+azp/posix/vm
```

Comment:
Please use
```suggestion
azp/posix/1
```
instead. `azp/posix/vm` is reserverd for tests that absolutely need a VM and don't really work in a container, which I don't think is the case here.

Reply:
done